pkg/testing/integration/program.go

```diff
@@ -1197,7 +1197,7 @@ func (pt *ProgramTester) TestPreviewUpdateAndEdits() error {
 }
 
 func (pt *ProgramTester) exportImport(dir string) error {
-	exportCmd := []string{"stack", "export", "--file", "stack.json"}
+	exportCmd := []string{"stack", "export", "--show-secrets", "--file", "stack.json"}
```

Comment:
Note that I think the important place we need to update is on line 1430 in performExtraRuntimeValidation.

I'm not positive we do have to update this location in `exportImport` - I think in this case we're directly roundtripping - so shouldn't matter whether we use `--show-secrets`.

Reply:
We won't need to update this one - you are correct!